app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
+# frozen_string_literal: true
+
+# rubocop:disable Lint/RedundantCopDisableDirective
+# rubocop:disable Style/Documentation
```

Comment:
Enabling/disabling these cops might not be necessary

Reply:
This more detailed description of the guidelines is very useful to me, thank you Mr @teekaytech, now I see better how to align myself to the project requirements, and also I'm going to take your word and address to a full time TSE because when I remove a # frozen_string_literal: true from a file and run rubocop next in the terminal, it says that a # frozen_string_literal: true is missing in that file, so I´m still confused with this problem. Here is a screen shot:

Reply:
![image](https://user-images.githubusercontent.com/63531796/107587965-63631080-6bc8-11eb-8f60-fb2d86096d53.png)